# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/shared/PFileSystemManager.ipdl
# Commit: b52f1a06e743
# Full Hash: b52f1a06e743ffb7695d349cc7edf11d0b5ce80e
# Author: Randell Jesup <rjesup@wgate.com>
# Date: 2022-09-10 21:45:26
# Regressor Bug: 1786465
# File Overlap Count: 1
# Description:
#   Bug 1786465 - Simplify FileSystemManagerChild by using the VirtualSendImpl attribute in PFileSystemManager definition; r=dom-storage-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D155352
# ==============================================================================

diff -r c4555c4937a9 -r b52f1a06e743 dom/fs/shared/PFileSystemManager.ipdl
--- a/dom/fs/shared/PFileSystemManager.ipdl	Sat Sep 10 04:35:48 2022 +0000
+++ b/dom/fs/shared/PFileSystemManager.ipdl	Sat Sep 10 04:35:49 2022 +0000
@@ -195,7 +195,8 @@
   /**
    * TODO: documentation
    */
-  async GetRootHandleMsg()
+  [VirtualSendImpl]
+  async GetRootHandle()
       returns(FileSystemGetHandleResponse response);
 
   /**
@@ -214,7 +215,8 @@
    *
    * @returns error or entry handle
    */
-  async GetDirectoryHandleMsg(FileSystemGetHandleRequest request)
+  [VirtualSendImpl]
+  async GetDirectoryHandle(FileSystemGetHandleRequest request)
       returns(FileSystemGetHandleResponse handle);
 
   /**
@@ -233,7 +235,8 @@
    *
    * @returns error or entry handle
    */
-  async GetFileHandleMsg(FileSystemGetHandleRequest request)
+  [VirtualSendImpl]
+  async GetFileHandle(FileSystemGetHandleRequest request)
       returns(FileSystemGetHandleResponse handle);
 
   /**
@@ -251,7 +254,8 @@
    *
    * @returns error or file object
    */
-  async GetFileMsg(FileSystemGetFileRequest request)
+  [VirtualSendImpl]
+  async GetFile(FileSystemGetFileRequest request)
       returns(FileSystemGetFileResponse response);
 
   /**
@@ -274,7 +278,8 @@
    *
    * @returns error or file system path
    */
-  async ResolveMsg(FileSystemResolveRequest request)
+  [VirtualSendImpl]
+  async Resolve(FileSystemResolveRequest request)
       returns(FileSystemResolveResponse response);
 
   /**
@@ -293,7 +298,8 @@
    *
    * @returns error or iterator
    */
-  async GetEntriesMsg(FileSystemGetEntriesRequest request)
+  [VirtualSendImpl]
+  async GetEntries(FileSystemGetEntriesRequest request)
       returns(FileSystemGetEntriesResponse entries);
 
   /**
@@ -310,7 +316,8 @@
    *
    * @returns error information
    */
-  async RemoveEntryMsg(FileSystemRemoveEntryRequest request)
+  [VirtualSendImpl]
+  async RemoveEntry(FileSystemRemoveEntryRequest request)
       returns(FileSystemRemoveEntryResponse response);
 
   /**
